host/vm-manager/0001-Support-for-HPD-in-Android.patch

```diff
@@ -0,0 +1,102 @@
+From 3f59bde2af597caec340cf4b4c05b0588e337dfc Mon Sep 17 00:00:00 2001
+From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
+Date: Wed, 5 Apr 2023 16:24:31 +0530
+Subject: [PATCH] Support for HPD in Android.
+
+Add support for dual display in vm-manager for Android.
+---
+ src/guest/guest.c         |  1 +
+ src/guest/start.c         | 27 +++++++++++++++++++++------
+ src/include/guest/guest.h |  5 +++++
+ 3 files changed, 27 insertions(+), 6 deletions(-)
+
+diff --git a/src/guest/guest.c b/src/guest/guest.c
+index 4d2bcf4..3982585 100644
+--- a/src/guest/guest.c
++++ b/src/guest/guest.c
+@@ -35,6 +35,7 @@ keyfile_group_t g_group[] = {
+ 	{ "mediation", { "battery_med", "thermal_med", "camera_med",NULL }},
+ 	{ "aaf",      { "path", "support_suspend", "audio_type", NULL } },
+ 	{ "guest_control", { "time_keep", "pm_control", NULL }},
++	{ "display",  {"no_of_displays", "connectors.0", "connectors.1", NULL } },
+ 	{ "extra",    { "cmd", "service", NULL } },
+ };
+ 
+diff --git a/src/guest/start.c b/src/guest/start.c
+index 25e1207..8f302e4 100644
+--- a/src/guest/start.c
++++ b/src/guest/start.c
+@@ -535,14 +535,29 @@ static int setup_sriov(GKeyFile *gkf, char **p, size_t *size)
+ 	int monitor_id = -1;
+ 	int vf = 0;
+ 	int mem = 0;
++	int number_of_displays = 1;
++	g_autofree gchar *connectors_0 = NULL;
++	g_autofree gchar *connectors_1 = NULL;
++
+ 	GError *error = NULL;
+ 	keyfile_group_t *g = NULL;
+ 
+ 	g = &g_group[GROUP_VGPU];
+ 	monitor_id = g_key_file_get_integer(gkf, g->name, g->key[VGPU_MON_ID], &error);
++
++	g = &g_group[GROUP_DISPLAY];
++	number_of_displays = g_key_file_get_integer(gkf, g->name, g->key[NO_OF_DISPLAYS], &error);
++	if(number_of_displays == 2) {
++		connectors_0 = g_key_file_get_string(gkf, g->name, g->key[CONNECTORS_0], &error);
++		connectors_1 = g_key_file_get_string(gkf, g->name, g->key[CONNECTORS_1], &error);
++	}
++
+ 	if (error) {
+ 		if (error->code == G_KEY_FILE_ERROR_KEY_NOT_FOUND) {
+-			cx = snprintf(*p, *size, " -display gtk,gl=on");
++			if (number_of_displays == 2)
++				cx = snprintf(*p, *size, " -display gtk,gl=on,connectors.0=%s,connectors.1=%s", connectors_0, connectors_1);
++			else
++				cx = snprintf(*p, *size, " -display gtk,gl=on");
```

Comment:
I think you can have 1 display, and still define the connector to direct the output to...
For example, the user can choose to output the single display to either HDMI or DP